schema.ts

```diff
@@ -2173,15 +2172,7 @@ function normalizeContentCategorization(
       return {type};
     }
     case ContentCategorizationType.Email: {
-      const {
-        type,
-        toProperty,
-        fromProperty,
-        subjectProperty,
-        htmlBodyProperty,
-        plainTextBodyProperty,
-        ...rest
-      } = value;
+      const {type, toProperty, fromProperty, subjectProperty, htmlBodyProperty, plainTextBodyProperty, ...rest} = value;
```

Comment:
For some of these I kind of like the old formatting of putting each value on a new line for easier diffs and blame history as these change, but the benefits of using prettier everywhere still seem bigger

Reply:
I'm supportive of that, to my knowledge that might be an eslint rule that controls that, I'd be happy to have it ported over here.